vts/trustedservices/README.md

```diff
@@ -2,3 +2,9 @@
 
 - `server-addr` (optional): address of the VTS server in the form
   `<host>:<port>`. If not specified, this defaults to `127.0.0.1:50051`.
+  Unless `listen-addr` is specified (see below), VTS server will extract the
+  port to listen on from this setting (but will listen on all local interfaces)
+- `listen-addr` (optional): The address the VTS server will listen on in the
+  form `<host>:<port>`. Only specify this if you want to restrict the server to
+  listen on a particular interface; otherwise, the server will listen on all
+  interfaces on the port specified in `server-addr`.
```

Comment:
IIUC, the `host` part in `server-addr` is ignored?

If so, should we call it `server-port` instead?

Reply:
It is ignored by the server, but not the client.

Reply:
Well, the assumption is only there if they share the config file, in which case, it's unavoidable. If it's not the case, you modify the config file for each consumer as appropriate.

Reply:
Effectively, this means that in a non-distributed deployment, you do not need to create multiple config file -- all services can be pointed to the same one, while also keeping the structure of config simple (i.e. just having `vts` rather than needing `vts-server` and `vst-client` sections that may then require duplicate values). 

If the deployment is non-distributed, the the view of the address space is necessarily shared. If the deployment is distributed, then (barring  the config being on network storage) each service will have its own instance of config file so the values can be adjusted as appropriate.

Reply:
just to be clear, in a split environment, is this configuration acceptable then:

* provisioning service
```
[...]
  server-addr: "vts.p-lb-pool.local:50051"
```

* verification service
```
[...]
  server-addr: "vts.v-lb-pool.local:50051"
```

* VTS
```
[...]
  listen-addr: ":50051"
```

Reply:
It is. Note that VTS could also alternatively be validly specified as
```
[...]
  server-addr: ":50051"
```